packages/management/framework/bundle/config/values.yaml

```diff
@@ -6,3 +6,5 @@ featureGatesPackageValues:
   namespace:
 tkrServicePackageValues:
   namespace:
+addonsManagerPackageValues:
+  namespace:
```

Comment:
can you add addons-manager default package values like [this](https://github.com/vmware-tanzu/tanzu-framework/pull/2066/files#diff-8efd7b9cae81cb70bbe7a34730a0a6612f9f9e92ce95161e685af92a22ebc563R20) here as well

Reply:
Fixed